### ISSUE TITLE
option to clear recycle bin and temp files older than 30 days

### ISSUE THREAD

**Issue by Goldmaster** [linuxmint/cinnamon#9715]
Hi there, this is more of a feature request than a bug, will you add an option in the privacy settings with the ability to empty the recycle bin and temp files, if they have been sitting there for 30 days? something like windows 10 storage sense feature? I know Gnome has a clear temp and recycle bin files that are more than 30 days old and would be useful to include in the cinnamon desktop environment, please.

**Comment by Lurux** [comment]
Agreed, although it's probably not a priority it would be nice to see some housekeeping option like these pop up in Cinnamon :D

**Comment by Lurux** [comment]
Still up to date

**Comment by Goldmaster** [comment]
Would need to test and see

**Comment by Lurux** [comment]
No option in Cinnamon or Nemo in 22.3. Would expect too see this option either in Nemo or in the privacy option alolg with "recent files"

**Comment by JosephMcc** [comment]
This is not currently planned. All feature requests now go here: https://github.com/orgs/linuxmint/discussions
